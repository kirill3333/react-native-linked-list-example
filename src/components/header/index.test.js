import React from 'react';
import renderer from 'react-test-renderer';

import PageHeader from './index';

it('renders correctly', () => {
  const tree = renderer.create(<PageHeader title="Contacts" />).toJSON();
  expect(tree).toMatchSnapshot();
});
