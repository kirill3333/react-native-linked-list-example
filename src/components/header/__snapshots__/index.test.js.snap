// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f9f7f9",
      "borderBottomColor": "#e6e6e6",
      "borderBottomWidth": 1,
      "flexDirection": "column",
      "height": 60,
      "justifyContent": "flex-end",
      "marginTop": -3,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#000",
        "fontSize": 18,
        "fontWeight": "bold",
        "paddingBottom": 15,
      }
    }
  >
    Contacts
  </Text>
</View>
`;
